dlpy/layers.py

```diff
@@ -2561,7 +2561,7 @@ class EmbeddingLoss(Layer):
     can_be_last_layer = True
     number_of_instances = 0
 
-    def __init__(self, name=None, distance="L2", margin=2, src_layers=None, **kwargs):
+    def __init__(self, name=None, margin=2, src_layers=None, **kwargs):
```

Comment:
we remove this as we dont use it?

Reply:
Yes, the hidden option should not be exposed. Yes, I ran the unit tests and will add a benchmark unit test for the layer.